src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1664,13 +1662,10 @@ static bool processSubscriptions
         cacheSemGive(__FUNCTION__, "update lastNotificationTime for cached subscription");
       }
     }
-
-    /* Release object created dynamically (including the value in the map created by addTriggeredSubscriptions */
-    trigs->attrL.release();
-    delete trigs;
   }
 
-  subs.clear();
+  releaseTriggeredSubscriptions(subs);
+
```

Comment:
Not sure if releaseTriggeredSubscriptions() does more things than just delete, but if this fixes leaks and no harness test is affected in the usual `make test`, then we can assume that everything is ok with the change (I guess).